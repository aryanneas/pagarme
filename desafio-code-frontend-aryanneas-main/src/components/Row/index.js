import styled from 'styled-components'

import { sizes } from '../../utils/responsive/index'

const Row = styled.div`
  @media screen and (min-width: ${sizes.md}) {
    display: flex;
  }

  word-break: break-word;
  display: flex;
`

export default Row
