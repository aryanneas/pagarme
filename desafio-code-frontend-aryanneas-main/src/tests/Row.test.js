import React from 'react'
import { render } from '@testing-library/react'

import Row from '../components/Row'

const text = 'Hello World!'

describe('Row component', () => {
  it('should render', () => {
    const { asFragment } = render(<Row />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('should render children correctly', () => {
    const { getByText } = render(
      <Row>
        <p>{text}</p>
      </Row>
    )

    getByText(text)
  })
})
