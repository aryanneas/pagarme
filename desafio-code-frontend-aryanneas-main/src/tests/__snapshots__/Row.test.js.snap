// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row component should render 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ goAumS"
  />
</DocumentFragment>
`;
